nttt/tidyup.py

```diff
@@ -104,15 +95,15 @@ def tidyup_translations(arguments):
     if len(files_to_update) > 0:
         print("About to tidy up files:")
         for file in files_to_update:
-            print(" - {}".format(file.replace(str(folder), "")))
+            print(" - {}".format(os.path.basename(file)))
 
         process_yn = input("Continue (y/n):")
         if process_yn.casefold() == "y":
 
             for source_file_path in files_to_update:
-                file_name = source_file_path.replace(str(folder), "")
+                file_name = os.path.basename(source_file_path)
 
-                output_file_path = str(output_folder) + file_name
+                output_file_path = os.path.join(output_folder, file_name)
```

Comment:
These two lines are much more robust than the previous version. However, it doesn't work correctly for the new projects that have markdown files in subfolders (for quizzes). E.g. if the source file path is foo/bar/nl/quiz1/question_1.md and the output_folder is bar, then output_file_path will be foo/bar/output/question_1.md instead of foo/bar/output/quiz1/question_1.md, like it would in the previous version (just by coincidence).

Reply:
Good catch!
Let me fix this coming week
A note to myself: this may become handy: https://stackoverflow.com/questions/8693024/how-to-remove-a-path-prefix-in-python

Reply:
@robertjank I believe that I fixed the issue with incorrect output file names for quizzes.

Also thanks to the awesome `run-tests.py` I discovered and fixed another issue with nested "--- feedback ---" tags. Details: https://github.com/wheleph/nttt/pull/60/commits/21f4bfd0730b12ea5cd7fef17909a9a878929dd4

Can you have another look at the PR?